openmetadata-ui/src/main/resources/ui/src/components/TagsInput/TagsInput.component.tsx

```diff
@@ -128,7 +128,6 @@ const TagsInput: React.FC<Props> = ({ tags = [], editable, onTagsUpdate }) => {
         onCancel={() => setIsEditTags(false)}
         onSelectionChange={handleTagSelection}
       />
-      {!editable && tags.length === 0 && <div>{NO_DATA_PLACEHOLDER}</div>}
```

Comment:
Any reason for removing this ?
This was kept if there is no permission to Edit and there are no tags.